packages/charting/src/chart-setup.jsx

```diff
@@ -37,6 +43,32 @@ const ConfigureChartPanel = props => {
     </div>
   );
 
+  const handleAlertDialog = (open, callback) => {
+    setAlertDialog(
+      {
+        alertDialog: open
+      },
+      callback
+    );
+    setOpen(false);
```

Comment:
I think here you should set setOpen(open) to make it more dynamic

Reply:
true. done!